CSS_Tasks/CSS_Advanced/index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="bg">
+
+<head>
+    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type" />
+    <title>WebGL</title>
+    <link href="stylesheet.css" rel="stylesheet">
+</head>
+
+<body>
+    <header class="header">
+        <button type="submit" class="button greenButton"> Вход </button>
+        <button type="submit" class="button greenButton"> Регистрация </button>
```

Comment:
You should follow one naming convention for classes, here you should use "green-button" or change the other names where you use camelCase . Also you can think for better name, for example "button-style" or something like that.

Reply:
I agree for the "dash-style". https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/ You may be interested to read a bit more about BEM.